server/routes/datesRoutes.js

```diff
@@ -0,0 +1,36 @@
+const dateParser = require('../services/Parser')
```

Comment:
i'd recomend splitting this file up. One file that has only the routes the other file contains the controllers for the routes that way you can do the following 

`router.get('/dates', someHandler);` which is easier to read and allows you to split up common logic to specific places (such as formatting)

Reply:
I don't understand this comment.